cmake/Platform/System/PlatformElementsManager.cmake

```
@@ -90,10 +90,16 @@ endfunction()
 
 function(find_platform_libraries)
 
+    # Add NO_SYSTEM_ENVIRONMENT_PATH to stop find_file from returning
```

Please remove the comment lines and replace them with something like this (You can also use replace it directly):
```suggestion
    # Avoid searching system's PATH as the SDK's general `Libraries` directory 
    # may be used instead of the platform's directory
```